source/trunk/Esf.sln

```diff
@@ -0,0 +1,65 @@
+﻿
```

Comment:
why do we have source and nested folder trunk? I would put the whole code in source and remove the prototype from the main source folder. Also we can keep prototypes in different branches outside the main source.